packages/runtime/src/SSRCacheProvider.tsx

```diff
@@ -1,15 +1,22 @@
-import React from "react";
+import React, { useMemo } from "react";
+import { Dependency, GlobalOptions } from "./utils";
 
-const Context = React.createContext({});
+type SSRCacheProviderValue = {
+  cache: Record<string, any>;
+  globalOptions?: GlobalOptions,
```

Comment:
`globalOptions` can be implemented by users as we support subscriptions. Also, changing this will generate recalculations in all the hooks, which may be a performance issue, so I'd consider global options an anti-pattern.

Let's go ahead and remove it.